pkg/services/featuremgmt/manager.go

```diff
@@ -152,17 +151,8 @@ func (fm *FeatureManager) GetFlags() []FeatureFlag {
 }
 
 func (fm *FeatureManager) HandleGetSettings(c *contextmodel.ReqContext) {
-	res := make(map[string]interface{}, 3)
-	res["enabled"] = fm.GetEnabled(c.Req.Context())
-
-	vv := make([]*FeatureFlag, 0, len(fm.flags))
-	for _, v := range fm.flags {
-		vv = append(vv, v)
-	}
-
-	res["info"] = vv
-
-	response.JSON(http.StatusOK, res).WriteTo(c)
+	// pkg/services/accesscontrol/scope_test.go
+	response.Err(fmt.Errorf("WHY DOES REMOVING THIS BREAK TESTS!!!!"))
```

Comment:
@ArturWierzbicki -- I am stumped, removing this function has my local + drone fail when running tests with:
```
# github.com/grafana/grafana/pkg/services/accesscontrol.test
/opt/homebrew/Cellar/go/1.19.5/libexec/pkg/tool/darwin_arm64/link: running clang failed: exit status 1
Undefined symbols for architecture arm64:
  "_github.com/grafana/grafana/pkg/api/response.wrap_handler", referenced from:
      _github.com/grafana/grafana/pkg/web.mwFromHandler in go.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

FAIL	github.com/grafana/grafana/pkg/services/accesscontrol [build failed]
FAIL
```

Reply:
that is very weird indeed, I wonder if it could be related to removing the import of `github.com/grafana/grafana/pkg/api/response` rather than the function itself, but I haven't been able to figure out how that might impact anything either

Reply:
That seems to have done the trick :)